### DISCUSSION TITLE
Collections for kids or not english languages

### DISCUSSION THREAD

**Discussion by aldfeu** [meisnate12/Plex-Meta-Manager#617] [Q&A]
Hi,

Thanks for the great job you've done. I love using Plexx Meta Manager. SO surprising that Plex has not yet investigated more this feature. It should be integrated in the platform, don't you think ?

Can you give me advices on how I can create collections oriented for kids. I would like to be able to analyze a subset oriented for kids mixing animes and movies with a ranking for kids...

My second question is more complex, do you think it is possible to create collection dedicated per languages. I have french and hindi movies, and if i mix them in a wwolrd wide movie collections, only US movies are displayed because they are lot more ranked. What do you recommend ?

thanks

**Comment by meisnate12** [comment]
You can limit collections by content rating and by original language
